### ISSUE TITLE
connection lost while connecting localhost with pylabrad

### ISSUE THREAD

**Issue by pppplin** [labrad/pylabrad#352]
Hi, i'm new to labrad I just installed pylabrad in Windows7, when I try labrad.connect('localhost'), the "connection lost" error occurred. 
Would you lost any possible reasons for this error? I noticed that there are two related issues, but is there anyway that I could still use pylabrad instead of moving to scalalabrad? Or scalalabrad is the recommended interface? Many thanks.

**Comment by zibrov-zlobin** [comment]
scalalabrad - is the labrad manager that controls all the servers
pylabrad is a python wrapper to connect to a labrad manager and communicate with it.
you need to install scalalabrad and pylabrad.

**Comment by pppplin** [comment]
Does it means that in order to connect to a labrad server, I need to install scalalabrad? I am using the sourceforge version, is it outdated?

**Comment by maffoo** [comment]
@pppplin, yes the sourceforge version of labrad is old and no longer supported. You need to run the [scalabrad](https://github.com/labrad/scalabrad) manager, and then as @zibrov-zlobin said both the servers and pylabrad clients will connect to the manager in order to communicate with each other.
